content.tex

```diff
@@ -90,30 +93,35 @@ \section{Alerting}
 
 \section{On-Call Shift}
```

Comment:
I was struggling to understand the wording here. I think there was just a lot in one sentence. Is it correct that ALL shifts must be authorized by HR? Or just the concept of these types of shifts?

Reply:
My understanding is that we need to get HR authorization ONCE for the concepts and sign off on the pay.  (Who knows what happens with union discussions...).  My interpretation of this document from Cristian was that is was the basis of getting HR approval overall, so that out of hours help can happen as needed, without approval per shift. Definitely something to ask Cristian.